client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -2,22 +2,19 @@ import Button from '@material-ui/core/Button';
 import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
-import { Typography } from '@material-ui/core';
 
 interface Props {
   linkTo: string;
-  asideText: string;
   btnText: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
```

Comment:
This seems like a misleading name for this component - it doesn't include a header necessarily, just a `Custom Auth Button` :).

Reply:
Has been removed.